server/src/index.jsx

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const app = express();
+const port = 3002;
+const bodyParser = require('body-parser');
+const db = require('../../database/index.js');
+
+app.get('/', (req, res) => res.send('CavaTable reviews module server is working!'));
+
+app.use(bodyParser.json());
```

Comment:
This is a middleware statement which should be before your routes. (app.gets and app.posts)